tests/unit/ByPropertyIdArrayTest.php

```diff
@@ -387,10 +382,10 @@ public function addProvider() {
 
 	/**
 	 * @dataProvider addProvider
-	 * @param array $objectsSource
-	 * @param object $object
-	 * @param int $index
-	 * @param array $objectsDestination
+	 * @param Claim[] $objectsSource
```

Comment:
This does not need to be `Claim`. Also, be careful to not cause conflicts with 3.0.x that  be avoided.